### ISSUE TITLE
Wish list itemL Vertex Colors

### ISSUE THREAD

**Issue by theo-armour** [spite/THREE.MeshLine#20]
![image](https://cloud.githubusercontent.com/assets/547626/16867209/6ff2f79e-4a25-11e6-83be-93672c8afd7b.png)

@spite 

It would be really great to have meshLines with vertex colors in [Analemma 3D]( https://ladybug-analysis-tools.github.io/ladybug-web/analemma-3d/analemma-3d-r18.html )

Any chance of this? Anything I could do to help?

**Comment by spite** [comment]
Good idea! Basically is adding the attributes for it, and reading it from the shader. I might take a look at it in a few weeks, or someone can take up the task :)

**Comment by spite** [comment]
I've just realised I've replied essentially the same in https://github.com/spite/THREE.MeshLine/issues/34. Keep tuned :/
